app/src/main/java/ru/yandex/praktikumchatapp/MainActivity.kt

```diff
@@ -71,7 +71,7 @@ fun ChatScreen(
     modifier: Modifier = Modifier
 ) {
     val viewModel = remember { ChatViewModel() }
-    val messagesList = viewModel.messages.observeAsState(emptyList())
+    val messagesList = viewModel.messages.collectAsState(emptyList())
```

Comment:
Чтобы не писать везде обращение к стейту как `messagesList.value`, можно воспользоваться проперти делегатом:
```suggestion
    val messagesList by viewModel.messages.collectAsState(emptyList())
```